specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/DataCollectorLogs.json

```diff
@@ -230,7 +230,7 @@
           "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
         }
       },
-      "description": "Contains details when the response code indicates an error."
+      "description": "Error details when response code indicates an error."
```

Comment:
You have to change all descriptions at once. Please check the error message of LintDiff CI task.